apis/authentication/v1alpha1/openidconnect_types.go

```diff
@@ -116,6 +116,14 @@ type OIDCAuthenticationSpec struct {
 
 	// +optional
 
+	// ExtraClaims, if specified, causes the OIDCAuthenticator to copy listed claims to the
+	// user Extra field.
+	// Claims will be converted to lower case and prefixed with "gardener.cloud/user/" before being copied.
+	// If any of the extra claims is not present in the token then the token will be rejected.
+	ExtraClaims []string `json:"extraClaims,omitempty"`
```

Comment:
`make manifests` should be ran one more time so that the new comment is reflected in the CDR templates.

Reply:
Done